tap_hubspot/hubspot.py

```diff
@@ -389,6 +389,40 @@ def get_companies(self):
             offset_key=offset_key,
         )
 
+    def get_companies_v2(
+        self, start_date: datetime, end_date: datetime
+    ) -> Iterable[Tuple[Dict, datetime]]:
+        filter_key = "hs_lastmodifieddate"
+        obj_type = "companies"
+
+        properties = self.get_object_properties(obj_type)
+
+        gen = self.search(
+            obj_type,
+            filter_key,
+            start_date,
+            end_date,
+            properties,
+        )
+
+        for chunk in chunker(gen, 10):
```

Comment:
you should be able to use 100 here as well, but do test it.

Reply:
Yes